client/my-sites/plans-features-main/index.jsx

```diff
@@ -325,6 +327,8 @@ class PlansFeaturesMain extends Component {
 
 		return (
 			<div className="plans-features-main">
+				<QueryPlans />
+				<QuerySitePlans siteId={ site.ID } />
```

Comment:
I'll go with lodash's `get` since the component still uses the `site` object as a prop to a couple of `PlanFeature` components 😕